cumulus/pallets/parachain-system/src/lib.rs

```diff
@@ -1404,8 +1404,8 @@ impl<T: Config> Pallet<T> {
 	///
 	/// The caller assumes that the pallet will accept regular outbound message to the sibling
 	/// `target_parachain` after this call. No other assumptions are made.
-	#[cfg(feature = "runtime-benchmarks")]
-	pub fn open_outbound_hrmp_channel_for_benchmarks(target_parachain: ParaId) {
+	#[cfg(any(feature = "runtime-benchmarks", feature = "test-helpers"))]
+	pub fn open_outbound_hrmp_channel_for_benchmarks_or_tests(target_parachain: ParaId) {
```

Comment:
how do you ensure this doesn't leak into production runtimes via feature leak?

Reply:
I know about https://doc.rust-lang.org/cargo/reference/resolver.html#feature-resolver-version-2, but it looks like an easy regression to introduce in the future for people not aware about this

Reply:
Yea, maybe just use `std` instead of adding a new feature.

Reply:
@ordian @ggwpez 
I just saw `feature = "test-helpers"` or `feature = "test-utils"` in other places,
because `#[cfg(any(feature = "runtime-benchmarks", test))]` didnt work for me.

What do you exactly suggest?
What do you mean by "use `std` instead" here?

Reply:
@ordian @ggwpez 
I think I got it, "test-helpers" is removed and changed to:
```
#[cfg(any(feature = "runtime-benchmarks", feature = "std", test))]
```

@ggwpez Is that what you mean?

Reply:
Yep. The `test` is not propagated and only available for the current crate, so its not that helpful.